build/docker/gaia-taskenv/Dockerfile

```diff
@@ -1,14 +1,7 @@
-from taskcluster/tester:0.3.6
+from taskcluster/tester:0.4.4
 maintainer Aus Lacroix <aus@mozilla.com>
 
-# install our custom git plugin
-copy git/target/doc/git-ci-checkout-pr.1 /usr/local/man/man1/git-ci-checkout-pr.1
```

Comment:
We're using tc-vcs, just like mozharness does with tester when running gaia tests on gecko builds.